src/app/contextBroker/contextBroker.cpp

```diff
@@ -602,15 +605,19 @@ PaArgument paArgs[] =
 
 
 
-#define API_V2                                                                                                   \
-  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      entryPointsTreat                          }, \
-  { "*",      EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,      badVerbAllFour                            }, \
-  { "GET",    ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      getEntities                               }, \
-  { "*",      ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,      badVerbGetOnly                            }, \
-  { "GET",    IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     getEntity                                 }, \
-  { "*",      IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,     badVerbGetOnly                            }, \
-  { "GET",    IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, getEntityAttribute                        }, \
-  { "*",      IENTATTR,  IENTATTR_COMPS_V2,    IENTATTR_COMPS_WORD, badVerbGetOnly                            }
+#define API_V2                                                                                     \
+  { "GET",    EPS,       EPS_COMPS_V2,         ENT_COMPS_WORD,       entryPointsTreat           }, \
```

Comment:
Perhaps that comment is more suited for ContextAttribute.cpp/Metadata.cpp ?
Where it is actually used.
Apart from the normal documentation I don't know really where this information fits (in the source code).

Perhaps we should have a short (not so short) explanation in the header of contextBroker.cpp - if we did, then this info would fit in right there. However, we don't have that right now ...